aws-cli/README.md

```diff
@@ -0,0 +1,72 @@
+# aws 
+
+This task performs operations on Amazon Web Services resources using `aws`.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/aws-cli/aws-cli.yaml
+```
+
+## Parameters
+
+- **SCRIPT**: The script of aws commands to execute e.g. `aws $1 $2` This will take
+ the first value and second value of ARGS as `s3` and `ls` (default: `aws $@`)
+- **ARGS**: The arguments to pass to `aws` CLI, which are appended 
+    to `aws` e.g. `s3 ls` ( default: `["help"]` ).
+
+
+## Workspace
+
+- **source**: To mount file which is to be uploaded to the aws resources, 
+    this should be mounted using any volume supported in workspace.
+
+
+## Secret
+
+AWS `credentials` and `config` both should be provided in the form of `secret`,
+ which is mounted as `volume` in the task.
+
+[This](./example/secret.yaml) example can be referred to create `aws-secrets`.
+
+
+## Usage
+
+After creating the task, you should now be able to execute `aws` commands by 
+specifying the command you would like to run as the `ARGS` or `SCRIPT` param. 
```

Comment:
is it possible to have a link here as well to docs on aws commands that users could potentially run?

Reply:
Sure. Will do that :wink: